data/gaTable.yaml

```diff
@@ -0,0 +1,396 @@
+- longName: Publications Warehouse
+  shortName: Pubs
+  accountName: pubs.er.usgs.gov
+  viewID: '20354817'
+  accountId: '10113280'
+  webPropertyId: UA-10113280-1
+  internalWebPropertyId: '22153976'
+  viewName: pubs.er.usgs.gov
+  timezone: America/Chicago
+  websiteUrl: http://pubs.er.usgs.gov
+  botFilteringEnabled: 'TRUE'
+  description: I am example of project description
+- longName: Water Quality Portal
```

Comment:
We probably want to manually add some formatting to make the "rows" stick out a bit better.